src/getHumanAge.test.js

```diff
@@ -1,8 +1,34 @@
+'use strict';
+
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
   test('should be declared', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should be 0 in human age', () => {
+    expect(getHumanAge(0, 0)).toStrictEqual([0, 0]);
+  });
+
+  test('should be 0, 0 in human age when cat age 14,dog age 14', () => {
+    expect(getHumanAge(14, 14)).toStrictEqual([0, 0]);
+  });
+
+  test('should be 1, 1 in human age', () => {
```

Comment:
```suggestion
  test('should be [1, 1] in human age', () => {
```

when? write clear and understandable names of your test cases